api/src/firstrun.py

```diff
@@ -104,96 +104,98 @@ def create_admin(admins):
             print(e)
             print("Something went wrong while creating the new user. Please try again.")
 
+def create_member(**kwargs):
+        try:
+            kwargs = {'pending_activation': False, 'price_level': PriceLevel.Normal.value, **kwargs}
+            member = member_entity.create(kwargs)
+
+            db_session.commit()
+        except Exception as e:
+            print(e)
+            print("Something went wrong while creating the new user. Please try again.")
```

Comment:
The `create_admin` method can get away with this because it has an outer loop. But here you'll just ignore the error and continue on like nothing happened. Perhaps not the best UX?